### ISSUE TITLE
Instructor Notification

### ISSUE THREAD

**Issue by owenskarri** [owenskarri/owenskarri.github.io#2]
The website links added to my site per our assignments
Here is my completed assignment @ianshea

**Comment by ianshea** [comment]
Hey Karri,

I received the notification, but I'm not seeing any commits beyond just the very basic site we setup. Is it possible you haven't sync'd your repository?

**Comment by owenskarri** [comment]
I guess I didn't know I was suppose to do that, I'll do that and resend to
you.

**Comment by owenskarri** [comment]
Okay I sync it, it should have the links and css rules
Here is my completed assignment @ianshea

**Comment by ianshea** [comment]
No problem, it's looking good.
You do have one issue with your unordered list. You're opening the list with a `<u>` tag. Which should be a `<ul>` tag.

A really easy fix! Nice work.

**Comment by owenskarri** [comment]
Of course, me and my eyes and my spelling, lol.  Getting old sucks!  I will
get that changed. I made my css rules big and obnoxious so I could see they
worked easily :-)

**Comment by owenskarri** [comment]
Are you going to post today's lecture notes in canvas?

Thanks,
Karri

**Comment by ianshea** [comment]
Yep! Posted on Canvas.
